src/main/java/topaz/ui/Ui.java

```diff
@@ -0,0 +1,150 @@
+        scanner = new Scanner(System.in);
+    }
+
+
+    /**
+     * Reads the next line of user input.
+     *
+     * @return The line of user input as a {@link String}.
+     */
+    public String readCommand() {
+        return scanner.nextLine();
+    }
+    public void welcome() {
+        showLine();
+        System.out.println(" Hello! I'm Topaz - I work in the IPC's Strategic Investment Department. "
+                + "Long time no see! How have you been? Clocking in a lot of overtime?");
+        System.out.println(" What can I do for you?");
+        showLine();
+    }
+
+    public void showUserIobError() {
+        System.out.println("Please enter valid information after the keyword!");
+        System.out.println("Try again!");
+    }
+
+    public void showMarkIobError(int index) {
+        System.out.println("Invalid index number: " + index + " Please refer to your list.");
+    }
+    public void showException(Exception e) {
+        System.out.println(e);
+        System.out.println("Try again!");
+    }
+
+    public void goodbye() {
+        System.out.println(" Ugh, why do they have to call me for such a trivial matter... Sorry, something just came up at work, I've got to get going!");
+        System.out.println(" Bye. Hope to see you again soon!");
+        scanner.close();
+    }
+
+    public void showHelp() {
+        String guide = """
+                    1. Adding Tasks
+                    1) ToDos: Tasks without specific dates/times.
+                    Command: todo <description>
+                    Example: todo borrow book
+                    Response: Confirms the addition and displays the task.
+                    
+                    2) Deadlines: Tasks that must be completed by a specific date/time.         
+                    Command: deadline <description> /by <date/time>
+                    Example: deadline return book /by Sunday
+                    Response: Confirms the addition and shows the deadline.
+                    
+                    3) Events: Tasks that start and end at specific date/times.         
+                    Command: event <description> /from <start date/time> /to <end date/time>
+                    Example: event project meeting /from Mon 2pm /to 4pm
+                    Response: Confirms the addition and shows the event details.
+                    
+                    2. Viewing All Tasks
+                    Command: list
+                    Response: Displays all tasks in your list with their types and statuses.
+                    
+                    3. Mark task status
+                    1) Mark task as done: mark <index of task in list>
+                    2) Mark task as undo: unmark <index of task in list>""";
+        System.out.println(guide);
+    }
+
+    public void showTimeParseException(DateTimeParseException dateTimeParseException) {
+        System.out.println("Invalid input date time: " + dateTimeParseException);
+    }

```

Comment:
Should there be a blank line between different methods to be consistent with the other parts of the code?